branchff

```diff
@@ -116,6 +116,18 @@ logrun mkdir -p $WORKDIR
 gitlib::sync_repo $K8S_GITHUB_SSH $TREE_ROOT || common::exit 1 "Exiting..."
 logrun cd $TREE_ROOT
 
+# Is the master branch in a reasonable state to fast forward over the branch?
+# The master is always left with a vX.Y.0-alpha.0 tag when a new branch is
+# created
+GIT_DESCRIBE=$(git describe --abbrev=0)
+if [[ ! $GIT_DESCRIBE == ^v${RELEASE_BRANCH/release-/}.0-alpha.0 ]]; then
+  logecho
+  logecho "$FATAL!  master branch has been tagged to a newer version" \
```

Comment:
That's the idea.  Thinking out loud. The master tree has been polluted with newer tags which make it a non-candidate for fast-forwards.  The master branch is only a valid fast-forward source when it's tagged with alpha.0.    We'll probably need/want to talk this through.